models/judge/judge_score_cache.go

```diff
@@ -0,0 +1,30 @@
+package judge_model
+
+import (
+	"time"
+
+	"github.com/oj-lab/oj-lab-platform/models"
+	problem_model "github.com/oj-lab/oj-lab-platform/models/problem"
+	user_model "github.com/oj-lab/oj-lab-platform/models/user"
+)
+
+// user contest problem summary score info
+type ScoreCache struct {
```

Comment:
Perfer `JudgeScoreCache` here, naming should not differed in `service` or `db` modules.